### ISSUE TITLE
Unable to expand nested objects inside the csv

### ISSUE THREAD

**Issue by kkbharath90** [endyourif/CsvWebApiExample#1]
I was able to get the output in csv format , but i am unable to iterate a nested object inside the csv .  For example, if i have a car object inside a vehicle class , i am getting output as "id,vehicaldetails,System.car(anonymous object)" and i am unable to expand that object further

**Comment by endyourif** [comment]
Great comment. I suggest that you enhance the example with recursion. When looping through the fields, if the field is an object, you would want to call the function again to output the list of fields on the child object.

You may also want to include a prefix so it would be Vehicle.fieldName.

**Comment by kkbharath90** [comment]
Yes i was trying to implement the same, but since it is a generalized class i am having issue with naming the properties , moreover when i try to loop through field i get object of type anonymous , so recursive calling is not working
